### PR TITLE
Update the getting started guide

### PR DESCRIPTION
**Pull request by oraNod**
Relates to https://github.com/ansible-community/community-team/issues/537

This PR updates the getting started in an effort to make it clear what the purpose of the getting started guide from the perspective of the novice persona.

### CODE REVIEW

**Review by Andersson007 — APPROVED** [review]

File: docs/getting_started_user.md [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
-This guide takes you through a simple example to get a first quick success with PROJECTNAME. 
+Projects should provide users with a getting started guide.
+The purpose of a getting started guide is to increase adoption and remove barriers to entry for your project.
+Users should be able to complete the steps in approximately five minutes.
 
-1. Install PROJECTNAME.
+Your project's getting started guide should give users a first quick success.
+In general, user should install the project, quickly verify the installation, and run one or two commands.
+The commands in your project's getting started guide should do something basic but illustrate your project's core functionality.
+If appropriate, consider provide sample configuration values and artifacts that allow users to run code locally.
```
Comment (samccann) [inline docs/getting_started_user.md]:
```suggestion
If appropriate, consider providing sample configuration values and artifacts that allow users to run code locally.
```

File: docs/getting_started_user.md [hunk 2]
```diff
@@ -1,19 +1,37 @@
...
 
-```
-pip install PROJECTNAME
-```
+> The following steps are intended to give an idea of a reasonable getting started guide with an example project named "ansible-project".
```
Comment (samccann) [inline docs/getting_started_user.md:12]: I like how the `>` makes the text different on a preview. Do you think it's worth doing that to the whole intro above or overkill?

**Review by samccann — COMMENTED** [review]
a nit and a question

File: docs/getting_started_user.md (hunk 2 above)
Comment (oraNod) [reply docs/getting_started_user.md:12]: I feel like that would be overkill.

**Review by samccann — APPROVED** [review]

**Pull request merged**